src/aiida_quantumespresso/parsers/parse_raw/pw.py

```diff
@@ -921,6 +923,23 @@ def parse_stdout(stdout, input_parameters, parser_options=None, parsed_xml=None)
     parsed_data['structure'] = structure_data
     parsed_data['trajectory'] = trajectory_data
 
+    # Double check to detect error messages if CRASH is found
+    if crash_file is not None:
+        lines = crash_file.split('\n')
+
+        for line_number, line in enumerate(lines):
+            # Compare the line to the known set of error and warning messages and add them to the log container
+            detect_important_message(logs, line)
```

Comment:
I think in older versions of QE, errors are written to _both_ the CRASH file and stdout. In those cases, you are duplicating the error messages in the container. This duplication was fixed in lines 919 and 920, but you are undoing that here.